de-lib/src/main/java/org/iplantc/de/client/services/impl/ToolServicesImpl.java

```diff
@@ -51,9 +53,10 @@ public void searchTools(FilterPagingLoadConfig loadConfig,
         SortInfo sortInfo =
                 Iterables.getFirst(loadConfig.getSortInfo(), new SortInfoBean("NAME", SortDir.ASC));
 
-        address += "search=" + URL.encodeQueryString(searchTerm)
-                   + "&sort-field=" + sortInfo.getSortField().toLowerCase()
-                   + "&sort-dir=" + sortInfo.getSortDir().toString();
+        address +=
+                "search=" + URL.encodeQueryString(searchTerm) + "&sort-field=" + sortInfo.getSortField()
+                                                                                         .toLowerCase()
+                + "&sort-dir=" + sortInfo.getSortDir().toString();
```

Comment:
This formatting looks a little odd to me.
I think I prefer the old formatting where each query param was on its own line.